sycl/test/basic_tests/accessor/atomic_zero_dimension_accessor.cpp

```diff
@@ -63,4 +63,4 @@ int main(int argc, char *argv[]) {
   std::cout << std::endl;
 
   return 0;
-}
\ No newline at end of file
+}
```

Comment:
I'm surprised to see dynamic `asserts` in this test (lines 56/57). Unrelated to this PR though.

Reply:
The test was added like this in #4458. We should probably be more careful when reviewing tests and perhaps we should do another round of tests review to refactor them, i.e.:
- move to llvm-test-suite anything which requires actual devices
- and/or rewrite that to unit-tests
- remove runtime checks from `-fsyntax-only` tests to simplify and cleanup them